.github/workflows/phpunit-pgsql.yml

```diff
@@ -33,13 +15,40 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+
+    outputs:
+      src: ${{ steps.changes.outputs.src}}
+
+    steps:
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
+        id: changes
+        continue-on-error: true
+        with:
+          filters: |
+            src:
+              - '.github/workflows/**'
+              - 'appinfo/**'
+              - 'lib/**'
+              - 'templates/**'
+              - 'tests/**'
+              - 'vendor/**'
+              - 'vendor-bin/**'
+              - '.php-cs-fixer.dist.php'
+              - 'composer.json'
+              - 'composer.lock'
+
   phpunit-pgsql:
     runs-on: ubuntu-latest
 
+    needs: changes
+    if: needs.changes.outputs.src != 'false'
+
     strategy:
       matrix:
-        php-versions: ['8.0']
-        server-versions: ['stable28']
+        php-versions: ['8.2']
+        server-versions: ['master']
```

Comment:
Is that expected @skjnldsv ?
It seems the changes are tailored for master and not stable28

Reply:
there is no `stable28` templates, only master, you need to adjust the automated changes :)